src/dmd/dsymbol.h

```diff
@@ -330,9 +330,7 @@ class WithScopeSymbol : public ScopeDsymbol
 class ArrayScopeSymbol : public ScopeDsymbol
 {
 public:
-    Expression *exp;    // IndexExp or SliceExp
-    TypeTuple *type;    // for tuple[length]
-    TupleDeclaration *td;       // for tuples of objects
+    RootObject *scopedArray;
```

Comment:
this is private in D, but this will need to be visible to GDC & LDC

Reply:
no, it's rather that I forgot to set it private in c++

Reply:
updated